tools/plugins/com.liferay.ide.server.ui/plugin.xml

```diff
@@ -582,5 +582,14 @@
         state="true">
         <description>%moduleProjectDecoratorDescription</description>
      </decorator>
+     <decorator
+        adaptable="true"
+        class="com.liferay.ide.server.ui.LiferayServerModuleStatusLabelDecorator"
```

Comment:
we already have a module decorator (that is what is making the plugins have our custom icon) we should add to existing decorator instead of adding a new noe